qiita_db/metadata_template/test/test_prep_template.py

```diff
@@ -280,6 +280,8 @@ def test_delitem(self):
 @qiita_test_checker()
 class TestPrepTemplate(TestCase):
     def setUp(self):
+        self.QCN = \
```

Comment:
self.QCN is a state variable used in base and derived classes to store the name of the column where all column names are stored. We may need this in a comment, because it's not apparent what it's for, if we first see it here.

Reply:
added some explanation